### DISCUSSION TITLE
Inference using NCS2

### DISCUSSION THREAD

**Discussion by rumanto00** [open-edge-platform/anomalib#551] [Q&A] [answered]
how to inference using NCS2 device?
is there any sample code how to do?

**Comment by samet-akcay** [comment]
@rumanto00, We apologize for the delay in responding. We somehow missed this. We haven't tested it on NCS2, so we can't say how it would perform there.

**Comment by djdameln** [comment, marked as answer]
Sorry for the late response. I hope you managed to deploy your models on NCS2 in the mean time. For future reference, a similar question has been answered [here](https://github.com/openvinotoolkit/anomalib/discussions/632#discussioncomment-3877127).
